db/migrate/20190604090631_create_external_urls.rb

```diff
@@ -0,0 +1,11 @@
+class CreateExternalUrls < ActiveRecord::Migration[5.0]
+  def change
+    create_table :external_urls, :id => :bigserial do |t|
+      t.references :refers_to, :polymorphic => true, :index => true, :type => :bigint
+      t.string  "url"
```

Comment:
nitpick: but we are using symbols:

`t.string  "url"`

Reply:
Agree with @lpichler here, can we use all symbols for the column names?

Reply:
I missed that. Fixed now.